### PR TITLE
[sweep:integration] feat (TS): add getTransformationFilesAsJsonString

### PR DESCRIPTION
**Pull request by DIRACGridBot**
Sweep #7840 `feat (TS): add getTransformationFilesAsJsonString` to `integration`.

Adding original author @chaen as watcher.

BEGINRELEASENOTES

*Core
FIX: Returning DISET responses with >100000000 bytes

*Transformation
NEW: Use getTransformationFilesAsJsonString for faster getTransformationFiles

ENDRELEASENOTES

### CODE REVIEW

**Review by chrisburr — DISMISSED** [review]
Needs to

* replace getTransfromationFiles with getTransfromationFilesAsJsonString
* deprecate getTransfromationFilesAsJsonString
* remove include_web_records flag

File: src/DIRAC/TransformationSystem/DB/TransformationDB.py [hunk 1]
```diff
@@ -634,12 +633,7 @@ def getTransformationFiles(
 
         resultList = [dict(zip(columns, row)) for row in res["Value"]]
 
-        result = S_OK(resultList)
-        if include_web_records:
-            webList = [[str(item) if not isinstance(item, int) else item for item in row] for row in res["Value"]]
```
Comment (chrisburr) [inline src/DIRAC/TransformationSystem/DB/TransformationDB.py]: Can you remove this from the tasks as well

**Comment by chrisburr** [comment]
Otherwise looks good

File: src/DIRAC/TransformationSystem/DB/TransformationDB.py [hunk 2]
```diff
@@ -909,7 +903,6 @@ def getTransformationTasks(
                     return res
             resultList.append(taskDict)
         result = S_OK(resultList)
-        result["Records"] = webList
```
Comment (chrisburr) [inline src/DIRAC/TransformationSystem/DB/TransformationDB.py:910]: isn't `webList` now an unused variable?

**Review by fstagni — APPROVED** [review]

**Pull request merged**